src/redux/appReduxer.js

```diff
@@ -0,0 +1,127 @@
+import {getFormate} from "../utils.js";
+
+const ADD_SESSION_LENGTH = 'ADD_SESSION_LENGTH'
+const ADD_BREAK_LENGTH = 'ADD_BREAK_LENGTH'
+const RESET = 'RESET'
+const START_TIMER = 'START_TIMER'
+const TIME_REDUCTION = 'TIME_REDUCTION'
+const PAUSE_TIMER = 'PAUSE_TIMER';
+
+let initializationState = {
+  breakLength: 5,
+  sessionLength: 25,
+  currentTimeSession: 1500,
+  currentTimeBreak: 300,
+  isProcess: false,
+  timerId: null,
+  currentMode: 'session'
+}
+document.title = 'Pomodoro Clock'
+
+export default (state = initializationState, action) => {
+  switch (action.type) {
+    case ADD_SESSION_LENGTH: {
+      if (
+        ((state.sessionLength > 1 && action.min < 0)|| (state.sessionLength < 60 && action.min > 0))
+        && !state.isProcess) {
+        return {
+          ...state,
+          sessionLength: state.sessionLength + action.min,
+          currentTimeSession: (state.sessionLength + action.min) * 60
+        }
+      }
+      return state;
+    }
+    
+    case ADD_BREAK_LENGTH: {
+      if (
+        ((state.breakLength > 1 && action.min < 0) || (state.breakLength < 60 && action.min > 0))
+        && !state.isProcess) {
+        return {
+          ...state,
+          breakLength: state.breakLength + action.min,
+          currentTimeBreak: (state.breakLength + action.min) * 60
+        }
+      }
+      return state;
+    }
+    
+    case RESET: {
+      document.title = 'Pomodoro Clock'
```

Comment:
Why ``document.title`` is here?

Reply:
Is this a bad practice?

Reply:
maybe use state and set title for the document in the component?